- It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is  the group significantly different (with respect to systolic blood pressure!)  from the regular population?

- Set up the hypothesis test.
- Write down all the steps followed for setting up the test.
- Calculate the test statistic by hand and also code it in Python. It should be # 4.76190. What decision can you make based on this calculated value?

In [43]:
from scipy import stats
import numpy as np

population mean: μ = 120

sample size : n = 100 

sample mean: x̅ = 130.1

Standard Deviation sample: σn = 21.21

### Two-Sided Test: Hypotheses 

H0: μ = 120 (The group is not significantly different from the rest of the population)

Ha: μ != 120 (The group is significantly different from the population)

### Test Statistic (t) [calculated by hand]

t = (130.1 - 120)/((21.21)/sqrt(100))
t = 10.1/2.121

t = 4.76


### Confidence Level 

95% -> significance level α = 0.025 (α = 0.05/2 because it´s a two-sided test)


### T-table Critical Value (tc)
for df = 100-1 = 99 and α = 0.025

tc = 1.984 


In [46]:
#Test Statistic (t) [using python]

sample_mean = 130.1
pop_mean = 120
sample_std_dev = 21.21

t = ( sample_mean - pop_mean ) / ( sample_std_dev / np.sqrt(100) )
print("Our t score is: {:.2f}".format(t))

Our t score is: 4.76


In [47]:
# T-table Critical Value (tc) [using python]

tc = +-stats.t.ppf(1-(0.05/2),df=99)
tc

-1.9842169515086827

### Result

t = 4.76 > +-1.98 =tc

t is larger than tc

The H0 can be rejected, because the group is significantly different from the population.